src/single-spa-react.js

```diff
@@ -102,13 +102,7 @@ function unmount(opts, props) {
   return Promise
     .resolve()
     .then(() => {
-      const domElementGetter = chooseDomElementGetter(opts, props)
-
-      if (!domElementGetter) {
-        throw new Error(`Cannot unmount react application '${props.appName || props.name}' without a domElementGetter provided in either opts or props`)
-      }
-
-      opts.ReactDOM.unmountComponentAtNode(getRootDomEl(domElementGetter));
+      opts.ReactDOM.unmountComponentAtNode(opts.domElement);
```

Comment:
Instead of letting the user change which domElement is returned from the domElementGetter between mount and unmount, we always unmount the react component that was mounted.

This fixes a bug where parcels do not always have a domElement passed to their unmount lifecycle, since it's assumed you can unmount a parcel from wherever it was mounted.